### ISSUE TITLE
Feature Request: rotate the fixed gear

### ISSUE THREAD

**Issue by stevepowell2000** [nfriend/inspirograph#54]
It would be cool if you could rotate the "fixed" item when the bar is selected. I was going to see if I could add the rotation class to the fixed item if/when it was a bar. Would that be doable?

**Comment by gatzke** [comment]
I agree.  The skinny gears should be able to rotate, just as the fixed gears should be able to move. 

Still, pretty amazing.  I think of the time I spent with pins trying to get those damn rings to do anything nice and neat and it always turned out terrible.  This is awesome!

**Comment by Frozenfire92** [comment]
+1

I was about to make an issue for this myself. It would be nice for the long fixed gears to be able to rotate, although everything mentioned above sounds great too
